packages/cli/src/zosjobs/download/download-output/Output.definition.ts

```diff
@@ -73,7 +73,18 @@ export const OutputDefinition: ICommandDefinition = {
                 "data conversion is performed using the file encoding specified.",
             type: "string",
             conflictsWith: ["binary", "record"]
-        }
+        },
+        {
+            name: "wait-for-active", aliases: ["wfa"],
+            description: "Wait for the job to enter ACTIVE status before completing the command.",
+            type: "boolean",
+            conflictsWith: ["wait-for-output", "directory"]
```

Comment:
I think `wait-for-output` should stay in the `conflictsWith` array, but should `directory` be removed? If a user is downloading output for a job, they might want to specify a directory - I don't believe the `wait-for` parameters will have an adverse effect when specified in combination with `directory`.